Setup/Uninstall.php

```diff
@@ -48,6 +48,7 @@ public function uninstall(SchemaSetupInterface $setup, ModuleContextInterface $c
         try {
             $setup->startSetup();
             UninstallerClient::uninstall();
+            $this->config->setIsMerchantActive(false);
```

Comment:
CI found a bug... if the user installs the module but never goes to the dashboard afterward, they will get this error when they try to uninstall: `An authorization token is required for NS8 requests`

Reply:
This has been addressed in the latest commit